Emby.Naming/Video/VideoListResolver.cs

```diff
@@ -128,7 +126,8 @@ public IEnumerable<VideoInfo> Resolve(List<FileSystemMetadata> files, bool suppo
                             .Except(extras)
                             .ToList();
 
-                        info.Extras.AddRange(extras);
+                        extras.AddRange(info.Extras);
+                        info.Extras = extras;
```

Comment:
can you explain?

Reply:
Same as below, can’t modify a readonlylist